package std190527_Exception_OverLoad_IO;

public class Main {
	public static void main(String[] args) {
//		Lct1Exception.play2();
		
//		try {
//			Lct1Exception.play3();			
//		}catch(Exception e) {
//			System.out.println("뭔지는 모르겠지만 예외지롱");
//		}
//		Lct1Exception.play4();
		
//		Lct2FrqExceptions.nullPointEx();
//		Lct2FrqExceptions.arrayIndexOutOfBoundsEx();
//		Lct2FrqExceptions.fileNotEx();
//		Lct2FrqExceptions.stringIndexOutEx();
//		Lct2FrqExceptions.numFormatEx();
//		Lct4FileIO.play1();
//		Lct4FileIO.play2();
//		Lct5FileRead.play1();
//		Lct6FileWrite.play1();
		Work1 w = new Work1();
		w.runFileIOSystem();

	}
}
